src/EditorFeatures/Test/InheritanceMargin/InheritanceMarginTests.cs

```diff
@@ -81,11 +81,11 @@ private static Task VerifyNoItemForDocumentAsync(string markup, string languageN
 
             for (var i = 0; i < sortedActualItems.Length; i++)
             {
-                VerifyInheritanceMember(testWorkspace, sortedExpectedItems[i], sortedActualItems[i]);
+                await VerifyInheritanceMember(testWorkspace, sortedExpectedItems[i], sortedActualItems[i]);
             }
         }
 
-        private static void VerifyInheritanceMember(TestWorkspace testWorkspace, TestInheritanceMemberItem expectedItem, InheritanceMarginItem actualItem)
+        private static async Task VerifyInheritanceMember(TestWorkspace testWorkspace, TestInheritanceMemberItem expectedItem, InheritanceMarginItem actualItem)
```

Comment:
Should `Task` returning functions have the `Async` suffix?

Reply:
They should yes. Need to update https://github.com/dotnet/roslyn/pull/51812.

Reply:
in tests we're all over the map.